packages/widget-message-meet/src/components/loading-screen/index.js

```diff
@@ -3,12 +3,15 @@ import classNames from 'classnames';
 import {FormattedMessage} from 'react-intl';
 
 import Spinner from '../spinner';
+import SparkLogo from '../spark-logo';
 import styles from './styles.css';
 
 export default function LoadingScreen() {
   return (
     <div className={classNames(`loading`, styles.loading)}>
-      <div className={classNames(`spark-logo`, styles.logo)} />
+      <div className={classNames(`spark-logo`, styles.logo)}>
+        <SparkLogo />
+      </div>
```

Comment:
I wasn't aware of the CDN issue. Will change the component to use the css method.